dev/deps/spark-deps-hadoop-2-hive-2.3

```diff
@@ -197,7 +197,40 @@ metrics-jmx/4.2.2//metrics-jmx-4.2.2.jar
 metrics-json/4.2.2//metrics-json-4.2.2.jar
 metrics-jvm/4.2.2//metrics-jvm-4.2.2.jar
 minlog/1.3.0//minlog-1.3.0.jar
-netty-all/4.1.68.Final//netty-all-4.1.68.Final.jar
+netty-all/4.1.71.Final//netty-all-4.1.71.Final.jar
+netty-buffer/4.1.71.Final//netty-buffer-4.1.71.Final.jar
```

Comment:
Whoa, this doesn't look right. netty-all is (probably?) a packaging of all these subcomponents. 
Something changes in how it's declared. We might need to depend just on pieces of netty 4.1.71?

And agree, I think you need Maven exclusions to match SBT

Reply:
> Whoa, this doesn't look right. netty-all is (probably?) a packaging of all these subcomponents.
Something changes in how it's declared. We might need to depend just on pieces of netty 4.1.71?

The change occurred after version 4.1.69, [Netty#11732 netty-all should not re-package jars](https://github.com/netty/netty/pull/11732).

@srowen Do you mean that we need to introduce only the sub modules we really use? I think it's OK

But we originally introduced all Netty modules before 4.1.68 through  use `netty-all-4.1.68.Final.jar`. If we introduce subsets instead, will it cause some user compatibility problems?

Reply:
Let me investigate if there is a better way to upgrade first

Reply:
Or we can give up the upgrade of this version. I found that there are many users in the Netty community are giving feedback on this problem and hope to provide `netty-allxx.Final.jar` again for upgrade.

Reply:
@srowen I read the discussion of  [Netty#11732 netty-all should not re-package jars](https://github.com/netty/netty/pull/11732) and the recent issues&prs in Netty, although many people have dissenting opinion to the current usage way, it seems that netty-all uber jar has not been restored until 4.1.72, and no one has plans to do this. Do we need to wait for a new version that provides uber jar before upgrading?

Reply:
We are not in hurry here because of the log4j fix went in to netty.
As I see in netty log4j-core was used only in test:
```
$ rg -A2 log4j-core
common/pom.xml
78:      <artifactId>log4j-core</artifactId>
79-      <scope>test</scope>
80-    </dependency>

pom.xml
706:        <artifactId>log4j-core</artifactId>
707-        <version>${log4j2.version}</version>
708-        <scope>test</scope>
```

Reply:
Thanks @attilapiros